plugins/jabra-file/tests/jabra-panacast-50-setup.json

```diff
@@ -0,0 +1,153 @@
+          ]
+        },
+        {
+          "Length": 9,
+          "DescriptorType": 4,
+          "InterfaceNumber": 3,
+          "InterfaceClass": 3,
+          "UsbEndpoints": [
+            {
+              "DescriptorType": 5,
+              "EndpointAddress": 130,
+              "Interval": 4,
+              "MaxPacketSize": 1024
+            },
+            {
+              "DescriptorType": 5,
+              "EndpointAddress": 2,
+              "Interval": 4,
+              "MaxPacketSize": 1024
+            }
+          ]
+        }
+      ],
+      "UsbEvents": [
+        {
+          "Id": "ReadProp:Key=BUSNUM",
+          "Data": "001"
+        },
+        {
+          "Id": "ReadAttr:Attr=uevent",
+          "Data": "MAJOR=189\nMINOR=57\nDEVNAME=bus/usb/001/058\nDEVTYPE=usb_device\nDRIVER=usb\nPRODUCT=b0e/3012/800\nTYPE=0/0/0\nBUSNUM=001\nDEVNUM=058"
+        },
+        {
+          "Id": "ReadProp:Key=DEVNUM",
+          "Data": "058"
+        },
+        {
+          "Id": "ReadProp:Key=BUSNUM",
+          "Data": "001"
+        },
+        {
+          "Id": "ReadAttr:Attr=uevent",
+          "Data": "MAJOR=189\nMINOR=57\nDEVNAME=bus/usb/001/058\nDEVTYPE=usb_device\nDRIVER=usb\nPRODUCT=b0e/3012/800\nTYPE=0/0/0\nBUSNUM=001\nDEVNUM=058"
+        },
+        {
+          "Id": "ReadProp:Key=DEVNUM",
+          "Data": "058"
+        },
+        {
+          "Id": "InterruptTransfer:Endpoint=0x02,Data=BQEAAEYCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==,Length=0x40",
+          "Data": "BQEAAEYCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+        },
+        {
+          "Id": "InterruptTransfer:Endpoint=0x82,Data=BQAAAEYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==,Length=0x40",
+          "Data": "BQABANgCABFKYWJyYSBQYW5hQ2FzdCA1MAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+        },
+        {
+          "Id": "InterruptTransfer:Endpoint=0x02,Data=BQEAAUYCAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==,Length=0x40",
+          "Data": "BQEAAUYCAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+        },
+        {
+          "Id": "InterruptTransfer:Endpoint=0x82,Data=BQAAAEYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==,Length=0x40",
+          "Data": "BQABAcwCAwU4LjAuNwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+        },
+        {
+          "Id": "InterruptTransfer:Endpoint=0x02,Data=BQEAAkYCEwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==,Length=0x40",
+          "Data": "BQEAAkYCEwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=="
+        },
+        {
+          "Id": "InterruptTransfer:Endpoint=0x82,Data=BQAAAEYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==,Length=0x40",
```

Comment:
> but our device has a "mode" in which it's endpoints change

Ohh, interesting! How do you trigger the mode where the EPs change please?

Reply:
It can be triggered by changing a setting in the device. The simplest way to do so is using our software Jabra Direct. I can instruct on what setting and what value it needs.
However devices in that mode are fairly "mutually exclusive" with fwupd so to speak. So an out of the box device would never be in this mode, but also it can be set in this mode only through our software, which makes having to use fwupd fairly redundant. I hope this makes sense.
I can instruct on how to change the setting if it's a matter of updating the emulation, but changing the setting "through" code for the sake of emulation is far from ideal

Reply:
> However devices in that mode are fairly "mutually exclusive" with fwupd so to speak

Got it, in which case ignore I think. Thanks.